src/scripts/sync.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
 set -x
-ORB_EVAL_FROM=$(circleci env subst "${ORB_EVAL_FROM}")
-ORB_EVAL_TO=$(circleci env subst "${ORB_EVAL_TO}")
-ORB_EVAL_ARGUMENTS=$(circleci env subst "${ORB_EVAL_ARGUMENTS}")
-ORB_EVAL_PROFILE_NAME=$(circleci env subst "${ORB_EVAL_PROFILE_NAME}")
+ORB_EVAL_FROM="$(circleci env subst "${ORB_EVAL_FROM}")"
+ORB_EVAL_TO="$(circleci env subst "${ORB_EVAL_TO}")"
+ORB_STR_ARGUMENTS="$(echo "${ORB_STR_ARGUMENTS}" | circleci env subst)"
+ORB_STR_PROFILE_NAME="$(circleci env subst "${ORB_STR_PROFILE_NAME}")"
 
-if [ -n "${ORB_EVAL_ARGUMENTS}" ]; then
-    set -- "$@" "${ORB_EVAL_ARGUMENTS}"
+if [ -n "${ORB_STR_ARGUMENTS}" ]; then
+    set -- "$@" "${ORB_STR_ARGUMENTS}"
 fi
 
-aws s3 sync "${ORB_EVAL_FROM}" "${ORB_EVAL_TO}" --profile "${ORB_EVAL_PROFILE_NAME}" "$@"
+aws s3 sync "${ORB_EVAL_FROM}" "${ORB_EVAL_TO}" --profile "${ORB_STR_PROFILE_NAME}" "$@"
```

Comment:
Do we want `set -x` for the whole file in this script? Maybe we should wrap just the command in it like in `copy.sh`